src/OSPSuite.UI/Binders/AxisBinder.cs

```diff
@@ -174,14 +174,15 @@ private void adjustAxisMinMax()
       private void setAxisRange(bool sideMarginsEnabled, Size diagramSize)
       {
          var axisWidthInPixel = Axis.AxisType == AxisTypes.X ? diagramSize.Width : diagramSize.Height;
+         _axisView.Logarithmic = Axis.Scaling == Scalings.Log;
```

Comment:
First problem was that the logarithmic property was not set before calculating the range. When going from log-linear that meant the 0.7 side margin was applied.

Because we responded to many events in the past, this was always calculated multiple times. After the first, this flag was set correctly at method entry.